docs/_spec/TODOreference/dropped-features/weak-conformance-spec.md

```diff
@@ -1,54 +0,0 @@
----
-layout: doc-page
```

Comment:
I think it's valuable to have precise rules in these pages or at least refer to them. Of course, many pages here do not have precise rules, but I'd like us to move in the general direction of more precise rules, not loss.

So I'd either keep this page, or link to the section on Harmonization in the spec, if that's feasible.

Reply:
We don't publish the spec yet, so I cannot link to it. But I re-introduced an explicit reference to the relevant sections of the spec. (note that the content of `APPLIEDreference` is not what gets published at https://docs.scala-lang.org/scala3/reference/ at the moment, so this does not change anything to the currently published documents).